hydrabooster/run-node.go

```diff
@@ -174,20 +177,23 @@ func printStatusLine(ndht int, start time.Time, totalpeers int64, uniqpeers uint
 }
 
 // RunSingleDHTWithUI ...
-func RunSingleDHTWithUI(path string, relay bool, bucketSize int) {
+func RunSingleDHTWithUI(path string, relay bool, bucketSize int) error {
 	datastore, err := levelds.NewDatastore(path, nil)
+
```

Comment:
```suggestion
```

Reply:
I believe a common pattern is to leave no space between the call and the if err != nil